templates/js/ai-bot/.vscode/launch.json.tpl

```diff
@@ -102,7 +102,7 @@
             "stopAll": true
         },
         {
-            "name": "Debug in Test Tool",
+            "name": "Debug in Test Tool (Preview)",
```

Comment:
Looks the README.md needs to be updated as well.

Reply:
Yes. I see Xiaofu's PR already did that. Let me depend on her works  :).